scripts/keras_benchmarks/models/cifar10_cnn_benchmark.py

```diff
@@ -0,0 +1,120 @@
+'''
+Original Model from keras/examples
+
+Train a simple deep CNN on the CIFAR10 small images dataset.
+GPU run command with Theano backend (with TensorFlow, the GPU is automatically used):
+    THEANO_FLAGS=mode=FAST_RUN,device=gpu,floatx=float32 python cifar10_cnn.py
+It gets down to 0.65 test logloss in 25 epochs, and down to 0.55 after 50 epochs.
+(it's still underfitting at that point, though).
+'''
+
+from __future__ import print_function
+import time
+import keras
+from keras.datasets import cifar10
+from keras.models import Sequential
+from keras.layers import Dense, Dropout, Activation, Flatten
+from keras.layers import Conv2D, MaxPooling2D
+from keras.utils import multi_gpu_model
+
+from model import BenchmarkModel
+from models import timehistory
+from gpu_mode import cntk_gpu_mode_config
+
+
+class Cifar10CnnBenchmark(BenchmarkModel):
+
+    # TODO(anjalisridhar): you can pass test name and sample type when creating
+    # the object
+    def __init__(self):
+        self._test_name = "cifar10_cnn"
+        self._sample_type="images"
+        self._total_time = 0
+        self._batch_size = 32
+        self._epochs = 2
+
+    def run_benchmark(self, keras_backend=None, gpu_count=0):
```

Comment:
You can access the Keras backend (as a string) via `keras.backend.backend()`. Is it necessary to include it here as a method argument?

For consistency with `multi_gpu_model`, please rename `gpu_count` `gpus`